pkg/trace/stats/aggregation.go

```diff
@@ -62,9 +64,9 @@ func getStatusCode(s *pb.Span) uint32 {
 }
 
 // NewAggregationFromSpan creates a new aggregation from the provided span and env
-func NewAggregationFromSpan(s *pb.Span, origin string, aggKey PayloadAggregationKey) Aggregation {
+func NewAggregationFromSpan(s *pb.Span, origin string, aggKey PayloadAggregationKey, extraAggregators map[string]struct{}) Aggregation {
```

Comment:
As there is no plan to add additional aggregators at the moment, it would be slightly more performant if this was just checking a boolean instead of building a map to pass in and having to do a hash lookup.